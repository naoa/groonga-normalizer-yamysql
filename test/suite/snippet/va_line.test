register normalizers/yamysql

table_create Entries TABLE_NO_KEY
column_create Entries title COLUMN_SCALAR ShortText
column_create Entries content COLUMN_SCALAR ShortText

table_create Tokens TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram --normalizer NormalizerYaMySQL
column_create Tokens entries_title COLUMN_INDEX|WITH_POSITION Entries title
column_create Tokens entries_content COLUMN_INDEX|WITH_POSITION Entries content

load --table Entries
[
{"title": "ヴァイオリン", "content": "このヴァイオリンは壊れている"}
]

select Entries \
  --output_columns 'title, snippet(content, \
  "バイオリン", "<span class=\\"w1\\">", "</span>", {"normalizer": "NormalizerYaMySQL"}), content' \
  --command_version 2 \
  --match_columns 'content' \
  --query "バイオリン"
