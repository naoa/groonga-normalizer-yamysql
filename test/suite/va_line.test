register normalizers/yamysql

normalize NormalizerYaMySQLUnicodeCI "ヴァヴィヴヴェヴォ"
