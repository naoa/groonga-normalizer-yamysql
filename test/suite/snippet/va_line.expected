register normalizers/yamysql
[[0,0.0,0.0],true]
table_create Entries TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Entries title COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
column_create Entries content COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
table_create Tokens TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram --normalizer NormalizerYaMySQL
[[0,0.0,0.0],true]
column_create Tokens entries_title COLUMN_INDEX|WITH_POSITION Entries title
[[0,0.0,0.0],true]
column_create Tokens entries_content COLUMN_INDEX|WITH_POSITION Entries content
[[0,0.0,0.0],true]
load --table Entries
[
{"title": "ヴァイオリン", "content": "このヴァイオリンは壊れている"}
]
[[0,0.0,0.0],1]
select Entries   --output_columns 'title, snippet(content,   "バイオリン", "<span class=\\"w1\\">", "</span>", {"normalizer": "NormalizerYaMySQL"}), content'   --command_version 2   --match_columns 'content'   --query "バイオリン"
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        1
      ],
      [
        [
          "title",
          "ShortText"
        ],
        [
          "snippet",
          null
        ],
        [
          "content",
          "ShortText"
        ]
      ],
      [
        "ヴァイオリン",
        [
          "この<span class=\"w1\">ヴァイオリン</span>は壊れている"
        ],
        "このヴァイオリンは壊れている"
      ]
    ]
  ]
]
